src/Youtube.php

```diff
@@ -148,10 +149,11 @@ public function getCommentThreadsByVideoId($videoId = null, $maxResults = 20, $o
      * @param string $order         Specifies the order in which the API response should list comment threads. Valid values are: time, relevance.
      * @param array $part           Specifies a list of one or more commentThread resource properties that the API response will include.
      * @param bool $pageInfo        Add page info to returned array.
+     * @param string $pageToken     The pageToken parameter identifies a specific page in the result set that should be returned.     
      * @return array
      * @throws \Exception
      */
-    public function getCommentThreads($channelId = null, $id = null, $videoId = null, $maxResults = 20, $order = null, $part = ['id', 'replies', 'snippet'], $pageInfo = false)
+    public function getCommentThreads($channelId = null, $id = null, $videoId = null, $maxResults = 20, $order = null, $part = ['id', 'replies', 'snippet'], $pageInfo = false, $pageToken = null)
```

Comment:
Looks good to me, just please update the tests accordingly

Reply:
@artin1982 I see this was fixed, can you also please update the tests accordingly in order to merge it for the next release ?